hw/ip/otbn/dv/model/otbn_model.cc

```diff
@@ -499,6 +499,11 @@ int OtbnModel::set_software_errs_fatal(unsigned char new_val) {
   return 0;
 }
 
+int OtbnModel::set_no_sec_wipe_chk(unsigned char new_val) {
```

Comment:
I'd be inclined to make `set_no_sec_wipe_chk` take no argument and just set the flag to `true` when it's called: after all, we're never going to call it to clear the flag.